tests/tokenizers/test_tokenizer.py

```diff
@@ -80,3 +84,49 @@ def test_load_tokenizer():
     )['attention_mask']
     attn_mask_key = [1, 1, 1, 1] + [0] * (tokenizer.model_max_length - 4)
     assert attention_mask == attn_mask_key
+
+
+@pytest.mark.parametrize(
```

Comment:
can you add a test for the actual function you are modifying please?

Reply:
@dakinggg modified `test_multi_turn_chat_slicing` to test for this as well